src/content/ui/navigation/index.md

```diff
@@ -50,16 +48,12 @@ Applications with simple navigation and deep linking requirements can use the
 `Navigator` for navigation and the [`MaterialApp.routes`][] parameter for deep
 links:
 
+<?code-excerpt "ui/navigation/lib/navigator_named_routes.dart (push-route)"?>
 ```dart
-@override
-Widget build(BuildContext context) {
-  return MaterialApp(
-    routes: {
-      '/': (context) => HomeScreen(),
-      '/details': (context) => DetailScreen(),
-    },
-  );
-}
+child: const Text('Open second screen'),
+onPressed: () {
+  Navigator.pushNamed(context, '/second');
+},
```

Comment:
The paragraph below this snippet says "Routes specified here are called named routes", but this snippet changed from showing where named routes are declared to how they are used. I would either change this back to showing the `routes:` declaration or updating the paragraph afterwards to say something like "calling pushNamed looks at the named routes that are declared..."

Reply:
Added a small change. Take a look when you have a moment.